### DISCUSSION TITLE
No dashboard after installation

### DISCUSSION THREAD

**Discussion by david-77** [teslamate-org/teslamate#2991] [Getting started] [answered]
Hi,

I have just installed Teslmate as described with docker-compose file, and after entering my tokens, I get a page without dashboards.
![image](https://user-images.githubusercontent.com/46008203/203552637-dc5df2cb-b136-4db1-b2f7-7a05f6f0f338.png)

I have seen another discussion on the same issue, answers ask to change Grafana's url, but I did not touch anything in the code, and the Grafana display prompt me to login (which login ?)

I search in the Teslamate docs site but there is only installation/maintenance technicals info, I found nothing about functional use of teslamate/grafana.

![image](https://user-images.githubusercontent.com/46008203/203553156-bb3cf6aa-3ed9-4603-8582-8510f8066074.png)

Thanks for your help

**Comment by cwanja** [comment, marked as answer]
What is the URL in your settings for dashboards and web app?

Login is admin:admin. If you have Grafana publicly exposed, you should change the login.

**Reply by david-77** [reply]
Thansk for the reply.
I could set up the grafana url in the Teslamate settings (at the bottom page), and now the dashboard link appears, thanks 👍 
Now i face another issue (blank/emply dashboard, error 500), I may create another discussion about it.
